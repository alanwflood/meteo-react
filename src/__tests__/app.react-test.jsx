import React from "react";
import renderer from "react-test-renderer";
import App from "../app";
import Layout from "../layouts";

describe("The main app", () => {
  it("should render to screen", () => {
    const component = renderer.create(<Layout />);

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
