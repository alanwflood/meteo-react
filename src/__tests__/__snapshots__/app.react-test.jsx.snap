// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The main app should render to screen 1`] = `
<div
  className="main-content"
>
  <form
    className="search"
    onSubmit={[Function]}
  >
    <div
      className=" search-bar"
      id="PlacesAutocomplete__root"
      style={Object {}}
    >
      <input
        autoComplete="off"
        className=" search-input"
        onBlur={[Function]}
        onChange={[Function]}
        onKeyDown={[Function]}
        placeholder="Enter a location"
        style={Object {}}
        type="text"
        value=""
      />
    </div>
    <button
      type="submit"
    >
      <span
        className="SVGInline"
        dangerouslySetInnerHTML={
          Object {
            "__html": "test-file-stub",
          }
        }
      />
    </button>
  </form>
  <div
    className="logo"
  >
    <h1>
      <div>
        Meteo
      </div>
      <small>
        Weather at a Glance
      </small>
    </h1>
    <div
      className="tagline"
    >
      Search for a location above to get your 5 day weather forecast!
    </div>
  </div>
</div>
`;
